db/migrate/20160107182205_add_bourbons_user_constraint.rb

```diff
@@ -1,5 +1,9 @@
 class AddBourbonsUserConstraint < ActiveRecord::Migration
-  def change
+  def up
     change_column :bourbons, :user_id, :integer, null: false
   end
+
+  def down
+    change_column :bourbons, :user_id, :integer
+  end
```

Comment:
This down method is just changing it to the same thing the `up` method is changing it to...? So the migration is doing nothing? It's also pretty bad to be changing migrations after they've already been committed into master...